pkg/asset/cluster/tfvars.go

```diff
@@ -378,6 +378,21 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+
+		instanceServiceAccount := ""
+		if installConfig.Config.CredentialsMode == types.PassthroughCredentialsMode {
+			var found bool
+			serviceAccount := make(map[string]interface{})
+			err := json.Unmarshal([]byte(sess.Credentials.JSON), &serviceAccount)
+			if err != nil {
+				return err
+			}
+			instanceServiceAccount, found = serviceAccount["client_email"].(string)
+			if !found {
+				return errors.new("could not find google service account")
```

Comment:
Sorry, I had a typo in my psudo-example... these should have been `errors.New(` with a capital N.